workspaces/marketplace/plugins/marketplace-common/package.json

```diff
@@ -50,6 +50,8 @@
   },
   "dependencies": {
     "@backstage/backend-plugin-api": "^1.0.1",
-    "@backstage/errors": "^1.2.4"
+    "@backstage/errors": "^1.2.4",
+    "knex": "^3.0.0",
```

Comment:
As discussed on slack: I think we shouldn't have knex as a dependency in our common package. Either we move that knex code directly into the marketplace-backend or we can create a node module for that code.

Reply:
I have moved the database code to marketplace-backend, but knex dependancy is needed here just for the aggregationApi types `Knex.QueryBuilder`